lib/meetup_bot/application.ex

```diff
@@ -25,6 +25,7 @@ defmodule MeetupBot.Application do
           crontab: [
             {"@reboot", MeetupBot.SyncManualEventsWorker},
             {"@hourly", MeetupBot.MeetupCacheWorker},
+            {"@hourly", MeetupBot.GDGCacheWorker},
```

Comment:
separated the GDG part that was previously with the `MeetupCacheWorker` into its own independent worker so if one source fails it doesn't affect the other

Reply:
This wont fix #51 because that exception would still happen. Although is correct that meetup worker will finish if they do not share the same job.